src/affinemap.jl

```diff
@@ -89,9 +100,21 @@ Type for representing affine maps with zero intercept.
 """
 struct LinearMap{T,U} <: AbstractAffineMap{T}
     A::U
-    LinearMap(A::AbstractMatrix) = new{eltype(A),typeof(A)}(A)
+    LinearMap{T,U}(A) where {T,U} = new{T,U}(A)
```

Comment:
I think this one is not even needed tbh, I think when you define the struct then constructors like this are always the default.